urldownloader.py

```diff
@@ -346,6 +413,11 @@ def execute(self, request: ServiceRequest) -> None:
                 if soup.title and soup.title.string:
                     http_result["title"] = soup.title.string
 
+                try:
+                    detect_open_directory(request, soup)
```

Comment:
This source.html should be the final page. The previous code was executing the detection on both the requested_url and the actual_url, in case they were different. After the deployment, we should keep an eye if there are examples we are missing as open directory.